content/source/docs/enterprise/install/installer.html.md

```diff
@@ -172,7 +172,7 @@ FROM ubuntu:xenial
 
 # Install software used by Terraform Enterprise.
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    sudo unzip daemontools git-core ssh wget curl psmisc iproute2 openssh-client redis-tools netcat-openbsd ca-certificates
+    sudo unzip daemontools git-core awscli ssh wget curl psmisc iproute2 openssh-client redis-tools netcat-openbsd ca-certificates
```

Comment:
Keep in mind this will install a rather old version of `aws` since the Ubuntu repositories haven't been updated to include a newer version.